ui/pages/create-account/new-account.component.js

```diff
@@ -16,7 +17,19 @@ export default class NewAccountCreateForm extends Component {
 
   render() {
     const { newAccountName, defaultAccountName } = this.state;
-    const { history, createAccount, mostRecentOverviewPage } = this.props;
+    const {
+      history,
+      createAccount,
+      mostRecentOverviewPage,
+      accounts,
+    } = this.props;
+
+    const accountsNames = accounts.map((item) => item.name);
+
+    const accountCreateButtonDisabled = () => {
+      return Boolean(accountsNames.includes(newAccountName));
```

Comment:
Includes is already `boolean` type, so we don't need to cast it.

Reply:
Fixed in c71199a60e2949308b7659f613fe02a48f46d79d.